### DISCUSSION TITLE
Circles

### DISCUSSION THREAD

**Discussion by jheubuch** [Traewelling/traewelling#2028] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
As a Träwelling user I'd like to share my private check-ins just with a couple of selected users/friends/followers. Sometimes I want to check into a connection but I don't want everyone to see it, but also I don't want to have a private profile.

**Describe the solution you'd like**
I'd like to have the opportunity to select "best friends" or a circle of users that can also see my private statuses.

**Comment by bendix-dev** [comment]
Theoretically the current "private profile" already works in a similar way to the proposed option. 
The problem is that you can't post anything publicly on a private profile. 
Maybe it would be an idea to rework this so that having a private profile only differentiates between having to approve followers or not and everything else is decided by which visibility the user sets?

**Comment by 0AgentSmith** [comment]
In the meantime, a feature called "trusted users" has been implemented, could we use that instead as a visibility setting for check-ins?
As in: a check-in that is just visible for trusted users
![temp](https://github.com/user-attachments/assets/51d97597-1bbe-43ae-86aa-c170fee1a396)

**Reply by sirlupusdev** [reply]
This is eight months old, but still, that'd be a great idea - I've seen multiple threads with different approaches regarding it as well though (e. g. Allowing the public visibility setting to be actually public for private profiles).

**Comment by MrKrisKrisu** [comment]
Closed, as #3832 is merged.
